src/test/scala/com/twitter/finagle/postgres/values/ValuesSpec.scala

```diff
@@ -138,6 +141,21 @@ class ValuesSpec extends Spec with GeneratorDrivenPropertyChecks {
       )
       "parse uuids" in test(ValueDecoder.uuid, ValueEncoder.uuid)("uuid_send", "uuid")
       "parse dates" in test(ValueDecoder.localDate, ValueEncoder.date)("date_send", "date")
+
+      "parse jsonb" in {
+        val json = "{\"a\":\"b\"}"
+        val createBuffer = () => {
+          ValueEncoder.jsonb.encodeBinary(JSONB(json.getBytes), Charset.defaultCharset()).get
+        }
+        val buffer = createBuffer()
+        val version = buffer.readByte()
+        val encoded = Array.fill(buffer.readableBytes())(buffer.readByte())
+        version must equal(1)
+        encoded must equal(json.getBytes)
+
+        val decoded = ValueDecoder.jsonb.decodeBinary("", createBuffer(), Charset.defaultCharset()).get()
```

Comment:
We should make `ValueDecoder.jsonb` implicit now, and have it be `ValueDecoder[JSONB]`.